apiary.apib

```diff
@@ -1191,7 +1191,7 @@ Returns a guide on how exactly you should register the external bucket. The text
         + Members
             + snowflake
             + bigquery
-    + path (required, array[string]) - Path to the schema. For Snowflake backend it's in a format ['database-name','schema-name'], for BigQuery ['bigquery-dataset-name']
+    + path (required, array[string]) - Path to the schema. For Snowflake backend it's in a format ['database-name','schema-name'], for BigQuery ['dataset-name']
```

Comment:
Přemjšlel jsem jestli tam nedat i projectId - aby to  bylo futureproof kdybysme pak uměli všechno udělat přes API nějak (třeba vrátit nějakou autorizační URL kterou to user potvrdí nebo tak). Ale asi je to blbost to teď komplikovat. A holt to budeme muset udělat BC potom